app/services/conditions_not_met.rb

```diff
@@ -2,19 +2,21 @@ class ConditionsNotMet
   include ActiveModel::Validations
   include ImpersonationAuditHelper
 
+  attr_reader :auth, :application_choice
+
   def initialize(actor:, application_choice:)
     @auth = ProviderAuthorisation.new(actor: actor)
     @application_choice = application_choice
   end
 
   def save
-    @auth.assert_can_make_decisions!(application_choice: @application_choice, course_option_id: @application_choice.current_course_option.id)
+    auth.assert_can_make_decisions!(application_choice: application_choice, course_option_id: application_choice.current_course_option.id)
 
-    audit(@auth.actor) do
-      ApplicationStateChange.new(@application_choice).conditions_not_met!
-      @application_choice.update!(conditions_not_met_at: Time.zone.now)
-      UpdateOfferConditions.new(application_choice: @application_choice).call
-      CandidateMailer.conditions_not_met(@application_choice).deliver_later
+    audit(auth.actor) do
+      ApplicationStateChange.new(application_choice).conditions_not_met!
+      application_choice.update!(conditions_not_met_at: Time.zone.now)
+      application_choice.offer.conditions.update_all(status: 'unmet')
```

Comment:
As with the other PR, can we use `update` so that auditing happens (when we eventually switch it on)